src/egui_skia.rs

```diff
@@ -87,25 +86,27 @@ impl EguiSkia {
         &mut self,
         input: egui::RawInput,
         run_ui: impl FnMut(&Context),
-    ) -> (Duration, egui::PlatformOutput) {
+    ) -> egui::PlatformOutput {
         let egui::FullOutput {
             platform_output,
             textures_delta,
             shapes,
-            repaint_after,
+            pixels_per_point: _,
+            // TODO: How to handle multiple outputs
+            viewport_output: _
         } = self.egui_ctx.run(input, run_ui);
 
         self.shapes = shapes;
         self.textures_delta.append(textures_delta);
 
-        (repaint_after, platform_output)
+        platform_output
     }
 
     /// Paint the results of the last call to [`Self::run`].
-    pub fn paint(&mut self, canvas: &mut Canvas) {
+    pub fn paint(&mut self, canvas: &Canvas) {
         let shapes = std::mem::take(&mut self.shapes);
         let textures_delta = std::mem::take(&mut self.textures_delta);
-        let clipped_primitives = self.egui_ctx.tessellate(shapes);
+        let clipped_primitives = self.egui_ctx.tessellate(shapes, self.pixels_per_point);
```

Comment:
Shouldn't it be possible to read the pixels_per_point value from the egui ctx itself? 

Seems like in egui_glow they just read it from the egui_ctx directly, I think we should also be able to do that? Then we don't have to update the api.

https://github.com/emilk/egui/blob/c8dd3dd01a7ea40b95925b8de11ecfa981eaa738/crates/egui_glow/src/winit.rs#L114

Reply:
That line seems to read `self.pixels_per_point` which in turn is defined in the EguiGlow "state", seems like the same thing I did here, or am I missing something? https://github.com/emilk/egui/blob/c8dd3dd01a7ea40b95925b8de11ecfa981eaa738/crates/egui_glow/src/winit.rs#L19

Reply:
Oh, you're absolutely right! No idea what I was thinking there